qCC/ccCommandLineCommands.cpp

```diff
@@ -1909,6 +1928,10 @@ bool CommandApplyTransformation::process(ccCommandLineInterface& cmd)
 	{
 		return cmd.error(QObject::tr("Failed to read transformation matrix file '%1'!").arg(filename));
 	}
+	if (inverse) {
+		cmd.print(QObject::tr("Transformation before inverse:\n") + mat.toString(6));
```

Comment:
```suggestion
		cmd.print(QObject::tr("Transformation before inversion:\n") + mat.toString(6));
```

Reply:
Did you want to display the inverted matrix as well maybe?

Reply:
In normal case the applied matrix shown, so i just needed to add this to show both of them.